siliconcompiler/core.py

```diff
@@ -81,6 +81,9 @@ def __init__(self, design, loglevel=None):
 
         self.schema = Schema(logger=self.logger)
 
+        self.register_package_source('siliconcompiler',
+                                     'python://siliconcompiler')
```

Comment:
Does this work if python is installed from pip?

Reply:
yes, just gets the location that siliconcompiler is installed to pip or git works

Reply:
Also if we call `register_sc_data_source(chip)` in every dependency why do we need this at all?